test/unittests/libweb3jsonrpc/jsonrpc.cpp

```diff
@@ -558,6 +558,8 @@ BOOST_AUTO_TEST_CASE(contract_storage)
         "505600a165627a7a72305820d8407d9cdaaf82966f3fa7a3e665b8cf4e6"
         "5ee8909b83094a3f856b9051274500029";
 
+    const string runtimeCode = compiled.substr(58);

```

Comment:
Is there an easy way to find this offset? or just disassemble and look for RETURN